processor/processor.go

```diff
@@ -660,10 +660,17 @@ func (proc *HandleT) getFailedEventJobs(response transformer.ResponseT, metadata
 		}
 
 		id := uuid.NewV4()
+		// marshal error to escape any quotes in error string etc.
+		marshalledErr, err := json.Marshal(failedEvent.Error)
+		if err != nil {
```

Comment:
We will end up not saving the failed event. Instead we should save it somehow. 

Why not save everything {"error" : string(failedEvent.Error)}